manim/mobject/numbers.py

```diff
@@ -139,3 +140,18 @@ class Integer(DecimalNumber):
 
     def get_value(self):
         return int(np.round(super().get_value()))
+
+
+class Variable(VDict):
+    def __init__(self, var, label):
+        var_label = TexMobject(label + "=")
+        self.var_tracker = ValueTracker(var)
+        value_tex = (
+            DecimalNumber(self.var_tracker.get_value())
+            .add_updater(lambda v: v.set_value(self.var_tracker.get_value()))
+            .next_to(var_label, RIGHT)
+        )
+        VDict.__init__(self, {"label": var_label, "value": value_tex})
+
+    def get_tracker(self):
+        return self.var_tracker
```

Comment:
Issue: it's unnecessary to have `get_*` methods in python. In fact, we are trying to get rid of them all over the codebase.

Suggestion: remove this method and change `var_tracker` to just `tracker`. I think the name is cleaner that way :) If a user needs the tracker instead of doing `some_tracker.get_tracker()` they will just need to do `some_tracker.tracker`.

Reply:
Agreed and changed